tests/atest/rules/tags/empty-tags/default_and_empty_tags.robot

```diff
@@ -0,0 +1,24 @@
+*** Settings ***
+Documentation  doc
+Default Tags  tag  othertag
+
+*** Test Cases ***
+Test
+    [Documentation]  doc
+    [Tags]
+    Pass
+    Keyword
+    One More
+
+Test With Tags
+    [Tags]    dummy
+    Log    2
+
+
+*** Keywords ***
+Keyword
```

Comment:
Extend tests for keywords: http://robotframework.org/robotframework/latest/RobotFrameworkUserGuide.html#user-keyword-tags

Reply:
Sure. But " but possible Force Tags and Default Tags setting do not affect them" so I will probably put "Consider using NONE.." part only for tags from test cases.